gatsby-node.js

```diff
@@ -44,7 +45,37 @@ const createHomePage = async (createPage) => {
   })
 }
 
-// TODO: add action for creating section pages
+// TODO: add action for creating section pages]
+const createSections = async (createPage, slug) => {
+  const sectionResp = await axios.get(`${BASE_URL}https://www.thedp.com/section/${slug}.json`)
+  const mostReadDPResp = await axios.get(MOST_READ_DP_API)
```

Comment:
@peterbaile if MOST_READ_DP_API and TOP_ARTICLES_API never change, couldn't we make helper functions that do axios.get(MOSTREAD) and the same for TOP_ARTICLES instead of doing this?